examples/proto_bench/timing.diff

```diff
@@ -0,0 +1,81 @@
+diff --git a/src/dtree.ml b/src/dtree.ml
```

Comment:
Why a diff under git?

Reply:
Te be able to modify code easily to time up functions.  They are called in the scripts to patch the code.

Reply:
Why not adding some functions in Console to memorize the time at some points in the code and print it (perhaps using the Timed module)? This code could be activated using a --time flag for instance.

Reply:
I can indeed.